Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -88,7 +88,7 @@ public bool CheckCapability(MixedRealityCapability capability)
             if (WindowsApiChecker.UniversalApiContractV8_IsAvailable)
             {
 #if WINDOWS_UWP
-                return ((capability == MixedRealityCapability.SpatialAwarenessMesh) && WindowsSpatialSurfaces.SpatialSurfaceObserver.IsSupported());
+                return capability == MixedRealityCapability.SpatialAwarenessMesh && WindowsSpatialSurfaces.SpatialSurfaceObserver.IsSupported();
```

Comment:
```suggestion
                return (capability == MixedRealityCapability.SpatialAwarenessMesh) && WindowsSpatialSurfaces.SpatialSurfaceObserver.IsSupported();
```

nit: can we leave one of those parenthesis for readability?